modules/ai/assets/js/editor/pages/form-layout/components/attachments/attachment-json.js

```diff
@@ -0,0 +1,34 @@
+import { Thumbnail } from './thumbnail';
+import PropTypes from 'prop-types';
+import { Skeleton } from '@elementor/ui';
+
+export const AttachmentJson = ( props ) => {
+	const attachment = props.attachments?.find( ( item ) => 'json' === item.type );
+
+	if ( ! attachment ) {
+		return null;
+	}
+
+	if ( ! attachment.previewHTML ) {
+		return <Skeleton
+			animation="wave"
+			variant="rounded"
+			width={ 60 }
+			height={ 60 }
+		/>;
```

Comment:
does `disabled` need to be passed also here?

Reply:
No